packages/manager/modules/billing/src/main/billing-main.controller.js

```diff
@@ -1,7 +1,16 @@
 export default class BillingMainCtrl {
   /* @ngInject */
-  constructor(coreConfig, isPayAsYouGoAvailable) {
+  constructor(coreConfig, isPayAsYouGoAvailable, $translate) {
     this.coreConfig = coreConfig;
     this.isPayAsYouGoAvailable = isPayAsYouGoAvailable;
+    this.$translate = $translate;
+  }
+
+  descriptionOfHeading() {
+    const getRegion = this.coreConfig.getRegion();
```

Comment:
Same as here:
https://github.com/ovh/manager/pull/6688/files#r865634536

Reply:
It's applied